health/warnings.go

```diff
@@ -146,7 +171,7 @@ var derpRegionErrorWarnable = Register(&Warnable{
 var noUDP4BindWarnable = Register(&Warnable{
 	Code:                "no-udp4-bind",
 	Title:               "Incoming connections may fail",
-	Severity:            SeverityHigh,
+	Severity:            SeverityMedium,
```

Comment:
This seems like an internal error that shouldn't ever happen, so SeverityHigh also seems fine.

Reply:
But whatever